src/Xamarin.Android.Build.Tasks/Tasks/GeneratePackageManagerJava.cs

```diff
@@ -103,10 +131,133 @@ public override bool Execute ()
 			MonoAndroidHelper.CopyIfChanged (temp, dest);
 
 			try { File.Delete (temp); } catch (Exception) { }
-			
-			try { File.Delete (temp); } catch (Exception) { }
+
+			AddEnvironment ();
 
 			return !Log.HasLoggedErrors;
 		}
+
+		static readonly string[] defaultLogLevel = {"MONO_LOG_LEVEL", "info"};
+		static readonly string[] defaultMonoDebug = {"MONO_DEBUG", "gen-compact-seq-points"};
+		static readonly string[] defaultHttpMessageHandler = {"XA_HTTP_CLIENT_HANDLER_TYPE", "System.Net.Http.HttpClientHandler, System.Net.Http"};
+		static readonly string[] defaultTlsProvider = {"XA_TLS_PROVIDER", "btls"};
+
+		void AddEnvironment ()
+		{
+			var environment = new StringWriter () {
+				NewLine = "\n",
+			};
+
+			if (EnableLLVM) {
+				WriteEnvironment ("mono.llvm", "true");
+			}
+
+			AotMode aotMode;
+			if (AndroidAotMode != null && Aot.GetAndroidAotMode (AndroidAotMode, out aotMode)) {
+				WriteEnvironment ("mono.aot", $"{aotMode.ToString ().ToLowerInvariant()}");
```

Comment:
Why the interpolated string for the value when the interpolated value *is* the value?

I wonder what the resulting IL is...